### ISSUE TITLE
asdf erlang returning only R13B03 on high sierra

### ISSUE THREAD

**Issue by kokolegorille** [asdf-vm/asdf-erlang#88]
The title says most. On a fresh High Sierra, with asdf 0.51

```
$ asdf plugin-add erlang https://github.com/asdf-vm/asdf-erlang.git
$ asdf list-all erlang
Downloading kerl...
R13B03
```

The plugin returns only R13

**Comment by Stratus3D** [comment]
I've been using asdf-erlang on high sierra for several months now and have not experienced this issue. Do you happen to have a custom `IFS` set in your shell? That could affect the way the list of versions are delimited.

**Comment by kylegraydev** [comment]
Same for me, used commit `d0860caa5c6ac940dc8bcda01401d497a175c1a5`

**Comment by aj-foster** [comment]
I was able to replicate this, and bisect it:

```
AJ ❯ ~ ❯ $ ❯ cd ~/.asdf/plugins/erlang/
AJ ❯ erlang ❯ (d0860ca...) ❯ $ ❯ asdf list-all erlang
# All versions, ending in...
21.0.7
21.0.8
21.0.9
21.1
AJ ❯ erlang ❯ (d0860ca...) ❯ $ ❯ git checkout 72bf68d7d91c946c89b8ce9137befdb9535d3ece
Note: checking out '72bf68d7d91c946c89b8ce9137befdb9535d3ece'.
  # ...
HEAD is now at 72bf68d Use kerl 1.8.5 for Mojave support.
AJ ❯ erlang ❯ (72bf68d...) ❯ $ ❯ asdf list-all erlang
Downloading kerl...
R13B03
```

I'm unfamiliar with the `IFS` you mentioned. 😬

**Comment by aj-foster** [comment]
By manually manipulating the `KERL_VERSION` variable in `bin/utils.sh`, I believe the issue was introduced in `kerl` version `1.8.3`. Version `1.8.2` is the last working version.

**Comment by Stratus3D** [comment]
Thanks @aj-foster ! You are right, I think we recently upgraded from kerl `1.8.2` to `1.8.5`. I was not aware of the change in the output, but I see locally that using the new version it doesn't print all the available versions out.

**Comment by Stratus3D** [comment]
Fixed by #89

**Comment by sirn** [comment]
Oops, sorry about this. I made the upgrade to 1.8.5 and didn't noticed they changed the `otp_releases` output 🙇
